### PR TITLE
Use hybrid app-localize-behavior

### PR DESCRIPTION
**Pull request by CodeBaboon**
This is a half-step (we should be using d2l-localize-behavior) but in lieu of that change I'd like to bump to the hybrid version of the existing dep in order to remove downstream resolutions.

### CODE REVIEW

File: test/d2l-user-switcher.js [hunk 1]
```diff
@@ -5,9 +5,17 @@ describe('d2l-user-switcher', function() {
 	var component;
 	var hasUsersTemplate;
 
+	var componentQuerySelector = function(selector) {
+		if (!Polymer.Element) {
+			return component.querySelector(selector);
```
Comment (CodeBaboon) [inline test/d2l-user-switcher.js]: It's not clear to me why the change I made (pulling in the newer app-localize-behavior) suddenly triggered this to no longer work in Polymer 2.

Comment (jstefaniuk-d2l) [reply test/d2l-user-switcher.js]: Use `component.$$(selector)` maybe and it may work for both?

Comment (CodeBaboon) [reply test/d2l-user-switcher.js]: Will give it a try, I'm just more confused why this cropped up in this PR specifically. Doesn't seem like bumping the `app-localize-behavior` should have suddenly changed this.

Comment (CodeBaboon) [reply test/d2l-user-switcher.js]: `component.$$(selector)` seems to work for both variants. However, according to https://www.polymer-project.org/2.0/docs/upgrade:
> The $$ method isn't available. Use this.shadowRoot.querySelector instead.

Even though it works, seems best to follow the docs here in case one day it does not. Will just leave as-is.
